### DISCUSSION TITLE
Quasar is now showing (This command is now reserved only for scaffolding with CUSTOM starter kits or legacy v0.x projects. ) i've been using quasar for sometime now and this is the first time quasar showing me this . is there any new changes with quasar cli or what??

### DISCUSSION THREAD

**Discussion by olelabs** [quasarframework/quasar#13551] [General - Components / Directives / etc] [answered]
quasar info output:

Operating System        Windows_NT(10.0.19044) - win32/x64
NodeJs                  14.16.0

Global packages
  NPM                   6.14.11
  yarn                  Not installed
  @quasar/cli           1.3.2
  @quasar/icongenie     2.5.1
  cordova               10.0.0

I'm using quasar cli , It should create a project when using command line quasar create.

Is there any new changes in quasar cli or what ??
![quasar_issue](https://user-images.githubusercontent.com/70916122/170682887-af197cd9-16e1-4023-bb3d-2d9662278139.PNG)

**Comment by MilosPaunovic** [comment, marked as answer]
[https://quasar.dev/start/release-notes](https://quasar.dev/start/release-notes)

Why not just following instructions from the console and run `npm init quasar`?

**Comment by metalsadman** [comment]
Yes `quasar create` is what you see in the suggestion now https://quasar.dev/start/quasar-cli#tl-dr
